roles/ceph-common/tasks/main.yml

```diff
@@ -64,24 +64,16 @@
   tags:
     - package-install
 
-- name: get ceph rhcs version
-  shell: rpm -q --qf "%{version}\n" ceph-common | cut -f1,2 -d '.'
+- name: get ceph version
+  shell: ceph --version | cut -f3 -d ' '
```

Comment:
Turns out, because Ubuntu uses `dash` when Ansbile invokes a shell, neither `PIPESTATUS` nor `-o pipefail` will work.  I could set the shell executable to `/bin/bash` but that means hard coding the executable path.  Another solution is to just get rid of the pipe and split out the version on the ansible side:

```
  - name: get ceph version
    command: ceph --version
    register: ceph_version

  - set_fact:
      ceph_version: "{{ ceph_version.stdout.split(' ')[2] }}"
```

Only downside to that is that `ceph_version.stdin` is used in a couple of other places and will have to be modified.  Does that work?